nncf/quantization/algorithms/accuracy_control/algorithm.py

```diff
@@ -134,18 +145,24 @@ def restore_accuracy(
                 validate the provided model.
             The function should return the value of the metric with the following meaning:
             A higher value corresponds to better performance of the model.
-        :return: The quantized model whose metric `final_metric` is satisfied the following condition
-
-            initial_metric - final_metric <= max_drop.
+        :return: The quantized model whose metric `final_metric` is satisfied
+            the maximum accuracy drop condition.
         """
+        if self.drop_type == DropType.ABSOLUTE:
+            max_absolute_drop = self.max_drop
+        elif self.drop_type == DropType.RELATIVE:
+            max_absolute_drop = self.max_drop * initial_metric
+        else:
+            raise ValueError(f"{self.drop_type} drop type is not supported.")
+        nncf_logger.info(f"Maximum absolute accuracy drop: {max_absolute_drop}")
 
         backend = get_backend(initial_model)
         algo_backend = get_algo_backend(backend)
 
         accuracy_drop = initial_metric - quantized_metric
-        nncf_logger.info(f"Accuracy drop: {accuracy_drop}")
+        nncf_logger.info(f"Absolute accuracy drop: {accuracy_drop}")
```

Comment:
Since the drop type may be `ABSOLUTE` or `RELATIVE`, it wouldn't be clear to the user what's going on, if the message will always says `Absolute...`

Reply:
done